### PR TITLE
MERC-1445 - add read_only_files defaults

### PR DESCRIPTION
**Pull request by mjschock**
https://jira.bigcommerce.com/browse/MERC-1445

### CODE REVIEW

**Comment by pedelman** [comment]
ping @mcampa @sherrybc @bc-ong

**Comment by pedelman** [comment]
LGTM 👍

**Comment by pedelman** [comment]
@saurabh-g

**Comment by saurabh-g** [comment]
👍

File: config.json [hunk 1]
```diff
@@ -195,6 +195,16 @@
     "spinner-borderColor-dark": "#989898",
     "spinner-borderColor-light": "#ffffff"
   },
+  "read_only_files": [
+    "/assets/scss/components/citadel",
+    "/assets/scss/components/foundation",
+    "/assets/scss/components/vendor",
+    "/assets/scss/vendor",
```
Comment (sherrybc) [inline config.json]: 🔤

**Pull request merged**